### DISCUSSION TITLE
Do I need to manually setup mason and mason-lspconfig?

### DISCUSSION THREAD

**Discussion by 0xDmtri** [VonHeikemen/lsp-zero.nvim#233] [Q&A] [answered]
Could anyone please help, do I need to manually do this in my config:

```
require("mason").setup()
require("mason-lspconfig").setup()
```

If yes does the order matter with Lsp-zero? Thanks!

**Comment by VonHeikemen** [comment, marked as answer]
You don't have to add that to your config, lsp-zero will do it for you if you call `.preset()`, like in [this example](https://github.com/VonHeikemen/lsp-zero.nvim#usage).

And yes, order does matter. `mason.nvim` should be configured before nvim-lspconfig. If you do need to configure mason.nvim, do it before lsp-zero.
